app/models/fs/box.rb

```diff
@@ -18,6 +18,8 @@
 class Fs::Box < Box
   DISABLED_MESSAGE_DRAFTS_IMPORT_KEYWORDS = ['(oblasť SPD)']
 
+  after_create { update(syncable: tenant.feature_enabled?(:fs_sync)) }
```

Comment:
tu podla mna chces before create a len to nastavit. nemusis to potom updatovat ani ukladat

Reply:
hej, upravila som